.github/workflows/ci.yml

```diff
@@ -348,15 +348,28 @@ jobs:
       - name: Edit .npmrc [link-workspace-packages=false]
         run: sed -i -E 's/link-workspace-packages=(deep|true)/link-workspace-packages=false/' .npmrc
 
+      # - name: Configure pnpm to ignore minimum release age for @clerk packages
+      #   if: ${{ steps.task-status.outputs.affected == '1' }}
+      #   run: |
+      #     echo "minimum-release-age-exclude=@clerk/*" >> .npmrc
+      #     pnpm config set minimum-release-age-exclude @clerk/*
```

Comment:
Should be removed?

Reply:
https://github.com/clerk/javascript/pull/6869